NMS-14352: Update terminology (#5006)
OpenNMS/opennms: docs/modules/operation/pages/admin/webui/dashboard.adoc

.Information displayed in the alarms component
image::dashboard/03_dashboard-alarms.png[]

1. Node: Node label of the node the alarm is associated.
2. Severity: Severity of the alarm.
3. UEI: Shows the UEI of the alarm.
4. Count: Number of alarms deduplicated by the reduction key of the alarm.
5. Last Time: Time for the last occurrence of the alarm.

> 6. Log Msg: The log message from the event which is the source for this alarm.
> It is specified in the event configuration file in `<logmsg />`

Reviewer: ```suggestion
6. Log Msg: The log message from the event definition.
```

.Information displayed in the Notifications component
image::dashboard/04_dashboard-notifications.png[]

> 1. Node: Label of the monitored node the notification is associated with.

Author: I'd lean towards "monitored" being implied. @mmahacek, do you think we can just say "node" here?

> 1. Node: Label of the monitored node the notification is associated with.

Reviewer: Totally.

image::dashboard/05_dashboard-outages.png[]

1. Node: Label of the monitored node with ongoing outages.

> 2. Current Outages: Number of services on the node with outages and total number of monitored services, e.g. with the natural meaning of "3 of 3 services are affected."

Reviewer 2: Looking at the sample graphic now, I think it's just a way to explain the absence of nouns in what is displayed. I think the phrase confuses things, so suggest the following:

```suggestion
2. Current Outages: Number of services on the node with outages out of the total number of monitored services; for example, "3 of 3 services are affected".
```

3. 24 Hour Availability: Availability of all services provided by the node calculated by the last 24 hours.

[[ga-dashboard-resource-graph-viewer]]
=== Resource Graph Viewer

The Resource Graph Viewer lets you navigate to time series data reports to give a quick entry point to diagnose performance issues.

> This viewer filters nodes based on the Surveillance View context and shows one graph report at a time.

Author: Cool - I think what we have should cover this. What do you think, @Bonrob2? :)
